src/things_mcp/fast_server.py

```
@@ -385,32 +382,26 @@ def add_new_project(
+            area_id=area_id,
             todos=todos
         )
-        
-        if not result:
-            return "Error: Failed to create project"
-            
-        return f"Successfully created project: {title}"
+        if not project_id:
+            return f"Error: Failed to create project: {title}"
+        invalidate_caches_for(["get-projects", "get-today", "get-upcoming", "get-anytime"])
+        return f"Successfully created project: {title} (ID: {project_id})"
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)